cmd/register/register.go

```diff
@@ -0,0 +1,58 @@
+package register
+
+import (
+	"bytes"
+	"os/exec"
+	"path/filepath"
+
+	initconfig "github.com/dymensionxyz/roller/cmd/config/init"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/spf13/cobra"
+)
+
+func RegisterCmd() *cobra.Command {
+	registerCmd := &cobra.Command{
+		Use:   "register",
+		Short: "Registers the rollapp and the sequencer to the Dymension hub.",
+		Run: func(cmd *cobra.Command, args []string) {
+			home := cmd.Flag(initconfig.FlagNames.Home).Value.String()
+			rollappConfig, err := initconfig.LoadConfigFromTOML(home)
+			initconfig.OutputCleanError(err)
```

Comment:
don't think it's a good name for this function as it also gets nil errors (I'd expect it to only get error objects)